pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -525,7 +525,7 @@ func (c *Controller) Validate(podName, podNamespace string, groupAddress net.IP,
 		ruleTypePtr = new(v1beta2.NetworkPolicyType)
```

Comment:
Should we change the return type to `*types.McastNPRuleInfo` and return nil, if not found?

Reply:
done